ci/build_wheel.sh

```diff
@@ -35,6 +35,14 @@ if [[ "${package_dir}" != "python/libcuvs" ]]; then
     )
 fi
 
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+if [[ ${RAPIDS_CUDA_MAJOR} == "12" ]]; then
```

Comment:
Can this be written as “not 11” if we expect future major versions to behave like 12?

Reply:
Ah, the way I was thinking this would work with future versions is that we would just remove the if conditional but the logic inside the block would still remain at the top-level scope. I'll rewrite it to "not 11" because the answer to your question is yes.